index.js

```diff
@@ -11,46 +11,57 @@ let total = 0;
 let counter = 0;
 const template = $('#hidden-template').html();
 
-for (var version of java_version) {
+function request(url) {
+  const xmlHttp = new XMLHttpRequest();
+  xmlHttp.open("GET", url, false);
+  xmlHttp.send(null);
+  return JSON.parse(xmlHttp.response);
+}
+
+// Dockerhub pulls
+let result = request("https://cors-anywhere.herokuapp.com/https://hub.docker.com/v2/repositories/adoptopenjdk?page_size=100")
```

Comment:
What does limiting to 100 mean?

Reply:
the API returns 15 results per page otherwise. 100 is large enough to cover us.